kogito-timer-springboot/README.md

```diff
@@ -15,11 +15,66 @@ This example shows
 * optionally use Job Service that allows to externalize time tracking to separate service and by that offload the runtime service
 	
 	
-Intermediate timer event
-<p align="center"><img width=75% height=50% src="docs/images/process.png"></p>
+* Intermediate timer event
```

Comment:
which bpmn file is that ? Maybe add it in parenthesis

Reply:
fixed @radtriste